### PR TITLE
addArticle_FeaturedArticle_ArticleAdded()

### PR DESCRIPTION
**Pull request by kshitijSharma2014**
I have added a test which creates featured article and then confirm its presence on the main site page.

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191   @javigomez   @gunjanpatel  please have a look

File: tests/system/webdriver/tests/content/ArticleManager0003Test.php [hunk 1]
```diff
@@ -269,5 +269,69 @@ public function frontEndEditArticle_ChangeArticleText_ArticleTextChanged()
 		$this->articleManagerPage->trashAndDelete($articleName);
 		$this->assertFalse($this->articleManagerPage->getRowNumber($articleName), 'Test Article should not be present');
 	}
+	
+	/**
+	*@test
```
Comment (javigomez) [inline tests/system/webdriver/tests/content/ArticleManager0003Test.php]: You must create the proper DocBlocks for Methods, please check the **Class Method DocBlocks and Functions DocBlocks** section at http://joomla.github.io/coding-standards/?coding-standards/chapters/docblocks.md and update the pull request

Comment (kshitijSharma2014) [inline tests/system/webdriver/tests/content/ArticleManager0003Test.php]: thank you @javigomez :)

I will update all the pulls accordingly with proper docblocks and will sniff all of the code again by tomorrow to check if I have not followed the coding style anywhere else.

**Comment by kshitijSharma2014** [comment]
@javigomez  I have added a line, please have see it.
thanks :)

**Pull request merged**